src/ontology/mondo-edit.obo

```diff
@@ -462878,6 +462873,19 @@ name: neurological muscular channelopathy due to a genetic ryanodine receptor de
 xref: Orphanet:98742 {source="MONDO:equivalentTo"}
 is_a: MONDO:0019119 {source="Orphanet:98742", source="https://orcid.org/0000-0001-5208-3432"} ! muscular channelopathy
 
+[Term]
+id: MONDO:1010000
+name: pythiosis
+def: "A granulomatous disease caused by the aquatic organism pythium insidiosum occurring rarely in humans. It is classified into three forms: ocular, cutaneous, and arterial." [MESH:D058968]
```

Comment:
@katiermullen, please add your orcid here in addition to MESH. This way we know you have tweaked the definition (ie it is not the exact same as MESH)

Reply:
Done!